spyderlib/widgets/findreplace.py

```diff
@@ -29,32 +30,36 @@
 def is_position_sup(pos1, pos2):
     """Return True is pos1 > pos2"""
     return pos1 > pos2
-    
+
 def is_position_inf(pos1, pos2):
     """Return True is pos1 < pos2"""
     return pos1 < pos2
 
+def is_a_var_char(c):
+    """Return True if c is a valid variable character, False otherwise"""
+    return c.isalnum() or (c == '_')
+MAX_SEARCH_LIMIT = 80  # Used in Find/Replace default word logic

```

Comment:
This could be inside `FindReplace` class as a class variable....

```python
class FindReplace(QWidget):
     """Find widget"""
     MAX_SEARCH_LIMIT = 80  # Used in Find/Replace default word logic
     STYLE = {False: "background-color:rgb(255, 175, 90);",
              True: ""}
     visibility_changed = Signal(bool)
```

and then use it like `self.MAX_SEARCH_LIMIT`